packages/core/lib/command-utils.js

```diff
@@ -187,7 +187,9 @@ const runCommand = async function (command, options) {
 const displayGeneralHelp = () => {
   const yargs = require("yargs/yargs")();
   commands.forEach(command => {
-    yargs.command(require(`./commands/${command}/meta`));
+    if (command !== "install" && command !== "publish") {
```

Comment:
maybe leave a comment here about why we're doing this

Reply:
Yes, that makes sense. I will add a comment.